src/features/user/Profile/components/RedeemModal.tsx

```diff
@@ -103,7 +101,7 @@ export default function RedeemModal({
   async function redeemCode() {
```

Comment:
This is not used any more, and can be removed.

Reply:
Please remove this completely, don't leave it commented. If you need to go back to check it out, version control holds the history of the file.